src/main/kotlin/no/nav/familie/klage/formkrav/FormUtil.kt

```diff
@@ -3,26 +3,53 @@ package no.nav.familie.klage.formkrav
 import no.nav.familie.klage.behandling.dto.PåklagetVedtakDto
 import no.nav.familie.klage.formkrav.domain.Form
 import no.nav.familie.klage.formkrav.domain.FormVilkår
+import no.nav.familie.klage.formkrav.domain.FormkravFristUnntak
 
 object FormUtil {
 
     fun ferdigUtfylt(formkrav: Form, påklagetVedtak: PåklagetVedtakDto) =
-        påklagetVedtak.harTattStillingTil() &&
-            alleVilkårBesvart(formkrav) &&
+            påklagetVedtak.harTattStillingTil() &&
+            alleVilkårTattStillingTil(formkrav) &&
             (alleVilkårOppfylt(formkrav) || friteksterUtfylt(formkrav))
 
     fun alleVilkårOppfylt(formkrav: Form): Boolean {
-        return formkrav.alleSvar().all { it == FormVilkår.OPPFYLT }
+        return formkrav.alleSvar().all { it == FormVilkår.OPPFYLT } ||
+               (alleVilkårOppfyltUntattKlagefrist(formkrav) && klagefristUnntakErValgtOgOppfylt(formkrav.klagefristOverholdtUnntak))
     }
 
-    fun alleVilkårBesvart(formkrav: Form): Boolean {
-        return formkrav.alleSvar().none { it == FormVilkår.IKKE_SATT }
+    private fun klagefristUnntakErValgtOgOppfylt(unntak: FormkravFristUnntak?) =
+            unntak !== null && klagefristUnntakOppfylt(unntak)
```

Comment:
Hvis man har default verdi for FormkravFristUnntak så unngår man nullsjekken her